### ISSUE TITLE
Expose more Asset Party settings (page description, thumbnail, tags, etc) to tool code.

### ISSUE THREAD

**Issue by Igrium** [Facepunch/sbox-issues#3728]
### For?

Asset Party

### What can't you do?

I'm trying to write a script to port all of the PolyHaven HDRIs to Asset Party (there's 600 of them; I'm *not* doing them manually), and I've gotten 90% of the way there. I can scrape the PolyHaven API, compile them into Source 2 assets, and auto-generate thumbnails in Blender.

The problem is I can't seem to edit the thumbnail, description, or tags from code. And because Asset Party runs on Blazer, there's no API I can use to push to it manually.

### How would you like it to work?

It would be nice to have a C# API to be used in the context of S&Box tools that allows you to modify arbitrary data within an asset party entry. Because this is in the editor, the user would already be authenticated, so there wouldn't be any issues with that.

### What have you tried?

The *only* other solution I can think of is to write a browser emulator that interfaces with the Blazer websocket system on Asset Party and somehow hack Steam's authentication servers to trick it into letting a bot authenticate with Steam on Asset Party, while managing to bypass the captchas and other other security measures they take.

Basically, I'd have to become a blackhat hacker.

### Additional context

I'm currently on a mission to port all the PolyHaven assets (CC0) to Asset Party, so that there can be a central repo of all of them before we inevitably get a bunch of shitty ports of all the same assets.

I've already got a fair number of them done: https://asset.party/polyhaven
